lib/dialects/oracledb/index.js

```diff
@@ -128,7 +129,7 @@ Client_Oracledb.prototype.acquireRawConnection = function() {
           });
         });
       };
-      const fetchAsync = function(sql, bindParams, options, cb) {
+      const fetchAsync = promisify(function(sql, bindParams, options, cb) {
```

Comment:
Does this function use any closure variables, or it can be moved outside to only be promisified once?

Reply:
This function take connection from outside